articles.tex

```diff
@@ -526,7 +526,9 @@
 \asection{Resignations}
 An Executive Board Member may resign their position by submitting in writing the reason for resignation to the House Chairman.
 The resignation will be read at the following House Meeting and become effective at that time.
-The office will become vacant and the selection process for a new member as described in section \ref{Appointment of an Interim Director} will begin at that time.
+The office will become vacant and the selection process for a new member as described in section \ref{Appointment of an Interim Director} will begin at that time. 
```

Comment:
Oh, this is showing up in the diff because there’s a space at the end of the line I believe.

Reply:
We can just make a nonsemantic after ballots if it works